runtime/fastly/builtins/cache-simple.cpp

```diff
@@ -0,0 +1,786 @@
+#include "cache-simple.h"
```

Comment:
There were pretty minor differences between the existing simple-cache.cpp, nice!

Reply:
Yeah the process is fairly straightforward at this point, will try to get a few similar subsystems together for the next batch.